Performance-Task/README.md

```diff
@@ -0,0 +1,34 @@
+# The Project
+
+## The Brief
+
+Our client wants to build up a subscriber list to market their new product. The client is paying for AWS and wants us to use it for this project. You are a full-stack engineer and your project manager has tasked you with finding a way to store all the subscriber's emails and creating a way to capture subscriber emails.
+
+The frontend team has come up with a couple of prototypes for the frontend form. You could either wire up one of their prototypes or roll your own!
```

Comment:
Great idea to provide the prototypes 👍 (since it's really not the focus of this course + it mimics the real-world work environment)